.github/workflows/staticcheck.yml

```diff
@@ -13,6 +13,6 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 1
-    - uses: dominikh/staticcheck-action@v1.2.0
+    - uses: dominikh/staticcheck-action@v1
```

Comment:
This should be in a separate PR.

Reply:
Moved this commit to https://github.com/gonum/gonum/pull/1868

Reply:
Thanks